src/libs/actions/Welcome.js

```diff
@@ -8,17 +8,60 @@ import * as Policy from './Policy';
 import ONYXKEYS from '../../ONYXKEYS';
 import NameValuePair from './NameValuePair';
 import CONST from '../../CONST';
+import createOnReadyTask from '../createOnReadyTask';
+
+const [isReady, setReady] = createOnReadyTask();
```

Comment:
Would `isReady` remain resolved if we logout and login with another account?

Reply:
This is a nice example for a static (libs) vs instance (component) based code

Reply:
Good point and nope, but I made this task resettable in another PR so let me see about that...